src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier color1 = new ColorSupplier();
+
+    public static String getRandomBall(String color1) {
```

Comment:
this method can't have arguments - its sole purpose is to generate random balls. Color (also random) should be supplied by colorSupplier